data/brands/tourism/hotel.json

```
@@ -82,6 +82,17 @@
         "tourism": "hotel"
       }
     },
+    {
+      "displayName": "Adagio",
+      "id": "adagio-612c69",
+      "locationSet": {"include": ["150"]},
```

Their [website](https://www.adagio-city.com/gb/destinations.shtml) shows they also have locations in the Middle East and Brazil
```suggestion
      "locationSet": {"include": ["001"]},
```